core/whistle_apps-1.0.0/src/whapps_call.erl

```diff
@@ -868,6 +870,25 @@ set_from_tag(FromTag, #whapps_call{}=Call) when is_binary(FromTag) ->
 from_tag(#whapps_call{from_tag=FromTag}) ->
     FromTag.
 
+-spec remove_custom_channel_var(ne_binary(), whapps_call:call()) -> whapps_call:call().
+remove_custom_channel_var(Key, #whapps_call{}=Call) ->
+    whapps_call_command:set(wh_json:from_list([{Key, <<>>}]), 'undefined', Call),
+    handle_ccvs_remove([Key], Call).
+
+-spec remove_custom_channel_vars(ne_binaries(), whapps_call:call()) -> whapps_call:call().
+remove_custom_channel_vars(Keys, #whapps_call{}=Call) ->
+    whapps_call_command:set(wh_json:from_list([{Key, <<>>} || Key <- Keys]), 'undefined', Call),
+    handle_ccvs_remove(Keys, Call).
+
+-spec handle_ccvs_remove(wh_json:object(), whapps_call:call()) -> whapps_call:call().
+handle_ccvs_remove(Keys, #whapps_call{ccvs=CCVs}=Call) ->
+    lists:foldl(fun(Key, C) ->
+        case props:get_value(Key, ?SPECIAL_VARS) of
+          'undefined' -> C;
+          Index -> setelement(Index, C, 'undefined')
```

Comment:
I don't like using setelement on a record (breaks the abstraction), but I'm not thinking of a better way to do this...so, just stating it for the record.